src/dripper/polkadot/PolkadotActions.ts

```diff
@@ -96,7 +96,9 @@ export class PolkadotActions {
   }
 
   async teleportTokens(dripAmount: bigint, address: string, parachain_id: string): Promise<DripResponse> {
-    logger.info("💸 teleporting tokens");
+    logger.info(
+      `💸 teleporting ${dripAmount} tokens to ${address} from ${this.account ? this.account.address.toString() : ""} to parachain ${parachain_id}`,
```

Comment:
I don't know if it is possible to simplify it like this

```suggestion
      `💸 teleporting ${dripAmount} tokens to ${address} from ${this.account?.address.toString() ?? ""} to parachain ${parachain_id}`,
```

Reply:
Will give it shot :)